frappe_pwa/www/pwa.js

```diff
@@ -139,8 +142,54 @@ if ('serviceWorker' in navigator) {
             });
         }
     }
-} else if(window.location.protocol !== 'https:') {
+} else if (window.location.protocol !== 'https:') {
     console.warn('[PWA] Requires secure HTTPS connection');
 } else {
     console.warn('[PWA] No Service Worker support on your device');
 }
+
+function show_pwa_alert(message, seconds = 7) {
```

Comment:
Please do not change naming conventions... in general, and especially not in the same file.